plugin/save_command.py

```diff
@@ -60,6 +61,47 @@ def _purge_changes_async(self) -> None:
                 break
 
 
+class SaveTasksRunner:
+    def __init__(
+        self, text_command: LspTextCommand, tasks: List[Type[SaveTask]], on_complete: Callable[[], None]
+    ) -> None:
+        self._text_command = text_command
+        self._tasks = tasks
+        self._on_tasks_completed = on_complete
+        self._pending_tasks = []  # type: List[SaveTask]
+        self._canceled = False
+
+    def run(self) -> None:
+        for Task in self._tasks:
+            if Task.is_applicable(self._text_command.view):
+                self._pending_tasks.append(Task(self._text_command, self._on_task_completed_async))
```

Comment:
It wasn't changed in this PR, but the uppercase `Task` is a bit inconsistent with the lines 113-115 in this file, where it is lowercase. I don't know what the convention is (or if a convention exists for it), but I would say it should either be lowercase here or uppercase in lines 113-115.